### PR TITLE
Adding pose estimation to the readme

### PR DESCRIPTION
**Pull request by BloodAxe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +243,20 @@ model = models.get("model-name", pretrained_weights="pretrained-model-name")
...
+<table class="tfo-notebook-buttons" align="left">
+    <td width="500">   
+        <a target="_blank" href="https://colab.research.google.com/drive/1NMGzx8NdycIZqnRlZKJZrIOqyj0MFzJE#scrollTo=3UZJqTehg0On">
+            <img src="./documentation/assets/SG_img/colab_logo.png" /> Pose Estimation Transfer Learning
+        </a>
+    </td>
+</table>
+</br></br>
+
+https://colab.research.google.com/drive/1NMGzx8NdycIZqnRlZKJZrIOqyj0MFzJE#scrollTo=3UZJqTehg0On
```
Comment (Louis-Dupont) [inline README.md:259]: I think this is not wanted

File: README.md [hunk 2]
```diff
@@ -667,7 +697,9 @@ Deci provides implementation for various datasets. If you need to download any o
 - [Coco](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/datasets/detection_datasets/coco_detection.py)
 - [PascalVOC 2007 & 2012](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/datasets/detection_datasets/pascal_voc_detection.py)
   
-  
+### Pose Estimation
+
+- [COCO](https://github.com/Deci-AI/super-gradients/blob/cadcfdd64e7808d21cccddbfaeb26acb8267699b/src/super_gradients/recipes/dataset_params/coco_pose_estimation_dekr_dataset_params.yaml)
```
Comment (Louis-Dupont) [inline README.md:702]: The other links map to `.py` file, and here to `.yaml` file, is that on purpose?

File: README.md [hunk 3]
```diff
@@ -637,7 +662,12 @@ Detailed list can be found [here](http://bit.ly/3GnJwgZ)
 - [DarkNet-53](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/darknet53.py)
 - [SSD (Single Shot Detector)](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/ssd.py) 
 - [YOLOX](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/models/detection_models/yolox.py)
-  
+
+
+### Pose Estimation
+
+- [DEKR-W32-NO-DC](super_gradients/recipes/coco2017_pose_dekr_w32_no_dc.yaml)
```
Comment (Louis-Dupont) [inline README.md:669]: Link not working

File: README.md [hunk 4]
```diff
@@ -121,6 +122,16 @@ pip install super-gradients
 ```
 
 ## What's New
+
+__________________________________________________________________________________________________________
+Version 3.1.3 (July 19, 2023)
+
+* [Pose Estimation Task Support](https://docs.deci.ai/super-gradients/documentation/source/PoseEstimation.html) - Check out fine-tuning [notebook example](https://colab.research.google.com/drive/1NMGzx8NdycIZqnRlZKJZrIOqyj0MFzJE#scrollTo=3UZJqTehg0On) 
+* Pre-trained modified [DEKR](super_gradients/recipes/coco2017_pose_dekr_w32_no_dc.yaml) model for pose estimation (TensorRT-compatible)
```
Comment (Louis-Dupont) [inline README.md:130]: Link not working

**Review by Louis-Dupont — COMMENTED** [review]
Looks great, left some minor notes

File: README.md (hunk 1 above)
Comment (BloodAxe) [reply README.md:259]: Why? What is wanted instead?

Comment (BloodAxe) [reply README.md]: Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2

File: README.md (hunk 2 above)
Comment (BloodAxe) [reply README.md:701]: Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2

File: README.md (hunk 4 above)
Comment (BloodAxe) [reply README.md]: Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2

File: README.md (hunk 3 above)
Comment (BloodAxe) [reply README.md]: Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2

**Comment by BloodAxe** [comment]
@Louis-Dupont thanks for double-checking the links!

**Review by Louis-Dupont — APPROVED** [review]
LGTM
